README.md

```diff
@@ -40,6 +40,16 @@ Check out live demo: [https://flotiq-nextjs-shop-2.netlify.app](https://flotiq-n
     FLOTIQ_API_KEY=YOUR FLOTIQ API KEY
     SNIPCART_API_KEY=YOUR SNIPCART API KEY
     ```
+   To add `SNIPCART API KEY` to your starter you need to log in to the website: https://snipcart.com/
+
+   After this you need to get into your account options and choose `API KEYS` line. 
```

Comment:
Maybe we can supply users with the direct link?
e.g https://app.snipcart.com/dashboard/account/credentials